Nero/src/NeroLeptons.cpp

```diff
@@ -94,7 +95,7 @@ int NeroLeptons::analyze(const edm::Event & iEvent)
 
         l.iso = chIso + nhIso + phoIso; 
         l.p4.SetPxPyPzE( el.px(),el.py(),el.pz(),el.energy());
-        l.tightId = int(isPassTight);
+        l.tightId = int(isPassTight)*2  + int(inPassMedium);
```

Comment:
There is a typo:
inPassMedium -> isPassMedium

Does the build test actually pick up the correct branch?